codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/middleware/RetryMiddleware.kt

```diff
@@ -23,10 +23,10 @@ class RetryMiddleware : MiddlewareRenderable {
     override val position = MiddlewarePosition.AFTER
 
     override fun render(model: Model, symbolProvider: SymbolProvider, writer: SwiftWriter, op: OperationShape, operationStackName: String) {
-        writer.write("$operationStackName.${middlewareStep.stringValue()}.intercept(position: ${position.stringValue()}, middleware: \$N(${middlewareParamsString(model, symbolProvider, op)}))", AWSClientRuntimeTypes.Core.RetryerMiddleware)
+        writer.write("$operationStackName.${middlewareStep.stringValue()}.intercept(position: ${position.stringValue()}, middleware: \$N(${middlewareParamsString()}))", AWSClientRuntimeTypes.Core.RetryerMiddleware)
     }
 
-    override fun middlewareParamsString(model: Model, symbolProvider: SymbolProvider, op: OperationShape): String {
+    private fun middlewareParamsString(): String {
```

Comment:
dont think we need a sepaate function now for all of these, think we can inline these strings, what do you think?